journal-of-mammalogy.csl

```diff
@@ -140,9 +140,11 @@
       </if>
       <else-if type="bill book chapter graphic legal_case legislation motion_picture paper-conference  song" match="any">
         <group prefix=". " delimiter=" ">
-          <label variable="page" form="short" text-case="capitalize-first" suffix=" "/>
-          <text variable="page"/>
-          <text term="in"/>
+          <group delimiter=" ">
+            <label variable="locator" form="short" text-case="capitalize-first"/>
```

Comment:
Ah, I now found an example from the style guide:

> Fahr, J. 2013.  Rhinolophus ziama.  Ziama horseshoe bat.  Pp. 355–356 in Mammals of Africa. 4. Hedgehogs, shrews, and bats (M. Happold and D. C. D. Happold, eds.).  Bloomsbury Publishing. London, United Kingdom.

Makes sense. Seems only applicable to chapters, though, not books.

Anyway, I'll revert the "locator" change.